OMCompiler/Compiler/SimCode/SimCodeUtil.mo

```diff
@@ -746,6 +763,37 @@ algorithm
   end try;
 end createSimCode;
 
+
+public function checkAliasVarsVariablity "Functions which checks and assigns the alias variables Variabilty in xml"
+  input SimCodeVar.SimVar simVar;
+  input SimCode.HashTableCrefToSimVar inSimVarHT;
+  output SimCodeVar.SimVar outsimVar=simVar;
+algorithm
+  outsimVar := match simVar
+  local
+    DAE.ComponentRef varName,name;
+    SimCodeVar.SimVar tempvar;
+  case SimCodeVar.SIMVAR(aliasvar = SimCodeVar.ALIAS(varName))
+    equation
```

Comment:
If the intention is algorithmic, I would suggest to use algorithm instead of equation